sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample10_RunningInAzureStack.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor.Samples.Infrastructure;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Storage.Blobs;
+
+namespace Azure.Messaging.EventHubs.Processor.Samples
+{
+    /// <summary>
+    ///   An example of running the Event Processor in the Azure Stack platform.
```

Comment:
Consider renaming all your "Azure Stack" references to "Azure Stack Hub". Azure Stack is a brand that refers to a family of products (Azure Stack Edge, Azure Stack Hubs, and Azure Stack HCI). We should refer to Azure Stack Hub which is the only platform on which Event Hubs runs besides Azure.

Reply:
Fixed it.